gunfolds/conversions.py

```diff
@@ -823,7 +826,10 @@ def encode_list_sccs(glist):
     first_graph = True
     for i, g in enumerate(glist):
         if first_graph:
-            SCCS = [s for s in strongly_connected_components(graph2nx(g))]
+            if scc_members:
```

Comment:
This comparison should be 
```suggestion
            if scc_members is not None:
```
see [here](https://peps.python.org/pep-0008/#programming-recommendations)

Reply:
change made and pushed